test/systemtests/cfg.json

```diff
@@ -0,0 +1,29 @@
+[
```

Comment:
createcfg.py takes care of creating json file on the fly. Changed at places in makefile as well.